UI/window-basic-settings-stream.cpp

```diff
@@ -278,6 +278,9 @@ void OBSBasicSettings::UpdateKeyLink()
 	} else if (serviceName.startsWith("Restream.io")) {
 		streamKeyLink =
 			"https://restream.io/settings/streaming-setup?from=OBS";
+	} else if (serviceName.startsWith("Luzento.com")) {
```

Comment:
Since there is only a single service being added, `startsWith` is unnecessary and you should use a direct comparison.

Reply:
The reason I did a `startsWith` here is that Luzento has plans to introduce SRT and HLS in the future, and this is future proof. But if you would still prefer it being exact equality I will make that change.

Reply:
I'd prefer it for now, it can be changed when the future services are added. As demonstrated by several other single-server services using `startsWith`, it is common for new service requests to copy and change an existing service and this makes it less likely they'll accidentally introduce the same issue.

Reply:
sounds good, I've changed it to `==`